webapp/components/PostCard/index.vue

```diff
@@ -131,10 +131,11 @@ export default {
 }
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
 .ds-card-image img {
   width: 100%;
-  max-height: 300px;
+  max-height: 2000px;
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-14T16:42:20Z" title="Monday, October 14th 2019, 6:42:20 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -131,10 +131,10 @@ export default {
 }
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
 .ds-card-image img {
   width: 100%;
-  max-height: 300px;
+  max-height: 2000px;
```


```suggestion
  max-height: 2000px;
  object-fit: contain;
```

this keeps the image from being distorted on the `create post` page (after cropping + confirming)